CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # OverReact Changelog
 
+## 1.15.1
+
+> [Complete `1.15.1` Changeset](https://github.com/Workiva/over_react/compare/1.15.0...1.15.1)
+
+__Dependency Updates__
+
+* over_react_test `^1.1.1` (was `^1.0.1`)
```

Comment:
Is this really worth calling out in the release notes since it's a dev dependency change? It doesn't affect consumers whatsoever